test/api/core/entrypoints/embedded/index.test.js

```diff
@@ -104,7 +104,7 @@ describe('lib/core/api/core/entrypoints/embedded/index', () => {
 
     entrypoint = new EntryPoint(kuzzle);
 
-    Object.defineProperty(entrypoint, 'log', {
+    Object.defineProperty(entrypoint, 'logger', {
```

Comment:
Unit tests were broken because they checked that the logger was never called... but sinon's spies were put on the wrong entrypoint property name, so the checks always passed :expressionless: